node/src/index-xpring.js

```diff
@@ -0,0 +1,77 @@
+const { 
+    PayIDClient, 
+    TransactionStatus, 
+    Wallet, 
+    XRPClient, 
+    XRPLNetwork, 
+    XpringClient 
+} = require("xpring-js");
+
+// The expected address of the gRPC server.
+const grpcURL = "test.xrp.xpring.io:50051";
+
+// A wallet with funds on testnet.
+const wallet = Wallet.generateWalletFromSeed(
+  "snYP7oArxKepd3GPDcrjMsJYiJeJB",
+  true
+);
+
+// The number of drops to send.
+const dropsToSend = "10";
+
+// The Pay ID to resolve.
+const payID = '$dev.payid.xpring.money/hbergren'
+
+// The network to resolve on. 
+const network = XRPLNetwork.Test
+
+async function main() {
+    console.log("\nUsing rippled node located at: " + grpcURL + "\n");
+    const xrpClient = new XRPClient(grpcURL);
+
+    console.log("Using network: " + networkToString(network) + "\n");
+    const payIDClient = new PayIDClient(network);
+
+    const xpringClient = new XpringClient(payIDClient, xrpClient);
+
+    console.log("Sending:");
+    console.log("- Drops "+ dropsToSend)
+    console.log("- To: " + payID);
+    console.log("- From: " + wallet.getAddress() + "\n");
+    const hash = await xpringClient.send(dropsToSend, payID, wallet)
+
+    console.log("Hash for transaction:\n" + hash + "\n");
+  
+    const status = await xrpClient.getPaymentStatus(hash);
+
+    console.log("Result for transaction is:\n" + statusCodeToString(status) + "\n");
+}
+
+function networkToString(network) {
+    switch (network) {
+        case XRPLNetwork.Dev:
+            return "Devnet"
+        case XRPLNetwork.Test:
+            return "Testnet"
+        case XRPLNetwork.Main:
+            return "Mainnet"
+        default:
+            return "Unknown Network"
+    }
+}
+
+function statusCodeToString(status) {
+    switch (status) {
```

Comment:
If you want to be _super clever_, you may be able to rewrite this:
https://www.typescriptlang.org/docs/handbook/enums.html#reverse-mappings

I think that would look like this:
```js
function statusCodeToString(status) {
  return TransactionStatus(status).toUpperCase() || "UNKNOWN"
}
```

Reply:
"Keep in mind that string enum members do not get a reverse mapping generated at all."

I have no idea why this is and it's a total pain.

Reply:
Isn't `TransactionStatus` a numeric enum though?

Reply:
https://github.com/xpring-eng/Xpring-JS/blob/87f065ae95b4c05181ca61fa3c888f45fa826205/src/transaction-status.ts